src/v2/controllers/FormController.js

```diff
@@ -109,18 +115,21 @@ export default Controller.extend({
         } else {
           this.showFormErrors(model, error);
         }
+      })
+      .finally(() => {
+        this.toggleFormButtonState(false);
```

Comment:
Some forms might not have any button in form button bar. I guess we can check for button's existence and then toggle here. Or Jquery handles it ? 

button && button.toggleClass('link-button-disabled', disabled);

https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.js#L141

Reply:
That's fine. jQuery handles the edge case.